.github/workflows/rust.yml

```diff
@@ -30,13 +30,24 @@ env:
   LLVM_CONFIG_PATH: llvm-config-10
 
 jobs:
+  required:
+    needs:
+    - test
```

Comment:
You also need to check the result of the needed jobs, skipped/cancelled jobs will also make it passed.

Reply:
Thx~